src/core/plugin.cpp

```diff
@@ -113,7 +113,7 @@ struct PluginManager::PluginManagerPrivate {
         fs::path resolved = resolver->resolve(filename);
 
         if (fs::exists(resolved)) {
-            Log(Debug, "Loading plugin \"%s\" ..", filename.string());
+            Log(Debug, "Loading plugin \"%s\" from \"%s\" ..", name, resolved.string());
```

Comment:
I don't think that's necessary.

Reply:
Ok - I plan to change it to use the full `resolved` path then. See what you think.